### DISCUSSION TITLE
Right way of extending the theme's fontFamily

### DISCUSSION THREAD

**Discussion by jgoux** [tw-in-js/twind#143] [Q&A] [answered]
Hello all,

I'm trying to setup `Inter` font with twind, but I also want to have the other default fonts as fallback.

I didn't find a plain object exported from twind that would contain the default theme.fontFamily.sans defined [here](https://github.com/tw-in-js/twind/blob/a1535ce5f8edbda33d53c90d92e57e6ced2744e7/src/twind/theme.ts#L363).

So I'm trying to use the `theme` helper but I'm unsure about what to give it as a context.

```
extend: {
  fontFamily: {
    sans: ["Inter var", ...theme("fontFamily.sans")(/* context? */)],
  },
},
```

Thanks for your help.

**Comment by sastan** [comment, marked as answer]
oh... That is currently not possible. But it should. Would the following code work for you (once implemented)?

```js
setup({
  theme: {
    extend: {
      fontFamily: (theme) => ({
        sans: ["Inter var", theme("fontFamily.sans")]
      })
    }
  }
})
```

**Reply by jgoux** [reply]
Yes it would be great 👍

**Reply by sastan** [reply]
Should work with v0.16.3

**Reply by jgoux** [reply]
Awesome, thanks for the reactivity! 👏
